src/app/features/errors/requesting-tab-closed-error-msg.tsx

```diff
@@ -0,0 +1,38 @@
+import { useOnTabClosed } from '@app/routes/hooks/use-on-tab-closed';
+import { Box, Flex, color, Text } from '@stacks/ui';
+import { useState } from 'react';
+import { FiAlertCircle } from 'react-icons/fi';
+
+export function RequestingTabClosedErrorMessage() {
```

Comment:
I think we are showing what we typically call a `Warning` not an `Error` message?